test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -149,26 +150,37 @@ instance Exception PactTestFailure
 -- ----------------------------------------------------------------------- --
 -- Keys
 
-testKeyPairs :: IO [SomeKeyPairCaps]
-testKeyPairs = do
-    let (pub, priv, addr, scheme) = someED25519Pair
-        apiKP = ApiKeyPair priv (Just pub) (Just addr) (Just scheme) Nothing
-    mkKeyPairs [apiKP]
+type ChainwebKeyPair
+    = (PublicKeyBS, PrivateKeyBS, Text, PPKScheme)
+
+testKeyPairs :: ChainwebKeyPair -> IO [SomeKeyPairCaps]
+testKeyPairs (pub, priv, addr, scheme) =
```

Comment:
Let's either ditch `addr` and hardcode to `Nothing` or make it `Maybe Text`. No need to cover it in CW

Reply:
We're not using this for anything significant